src/features/auth/sign-in/components/Logo/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+const Logo = () => {
+  return (
+    <View>
+      <Text>My Custom Logo</Text>
+    </View>
+  );
+};
```

Comment:
```suggestion
const Logo = () => (
    <View>
      <Text>My Custom Logo</Text>
    </View>
  );
```

You can use implicit return if you want